release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
```

Comment:
I like the idea of starting the bullet points with action verbs, it effectively uses the parallelism prinicples as well as makes the sentences active. But please note the new features for the other sections in the OCP RN are written in a particular format. Most of them use You can now/With this release. For consistency, I suggest we add a stem sentence for the bullet points saying: With this release, you can: